compose/wordpress.md

```diff
@@ -105,9 +117,12 @@ minutes before the first load.
 ![WordPress Welcome](images/wordpress-welcome.png)
 
 ### Shutdown/Clean up
+
 `docker-compose down` will remove the containers and default network, but
```

Comment:
Maybe "The <command> command removes the containers and default network, but..."

I don't think that using an actual command as the noun of the sentence is a good idea, especially for non-native English speakers trying to parse the sentence.